### PR TITLE
feat(suite-native): updated AlertBox visual with versions

### PR DESCRIPTION
**Pull request by vytick**
## Description

Add multiple variants to AlertBox

Screenshot 2023-10-30 at 21 22 36
Screenshot 2023-10-30 at 21 05 08

### CODE REVIEW

File: suite-native/atoms/src/AlertBox.tsx [hunk 1]
```diff
@@ -1,46 +1,101 @@
...
+                color={variantToColorMap[variant].contentColor}
+                size="mediumLarge"
+            />
+            <Text
+                color={variantToColorMap[variant].contentColor}
+                variant="label"
+                style={applyStyle(textStyle)}
+            >
+                {title}
+                {textAppendix && ' '} {textAppendix}
```
Comment (Nodonisko) [inline suite-native/atoms/src/AlertBox.tsx]: What's are benefits of this? I think using just `title` should be enough

File: suite-native/atoms/src/AlertBox.tsx [hunk 2]
```diff
@@ -1,46 +1,101 @@
...
+
+type AlertBoxStyle = {
+    backgroundColor: Color;
+    contentColor: Color;
+};
 
 type AlertBoxProps = {
-    title: string;
-    isIconVisible?: boolean;
+    title?: string;
```
Comment (Nodonisko) [inline suite-native/atoms/src/AlertBox.tsx]: Better use React.ReactNode because of Translations.

**Review by Nodonisko — CHANGES_REQUESTED** [review]

File: suite-native/atoms/src/AlertBox.tsx (hunk 1 above)
Comment (vytick) [reply suite-native/atoms/src/AlertBox.tsx]: using Reactnode for title should probably solve this usecase. good point

File: suite-native/atoms/src/AlertBox.tsx (hunk 2 above)
Comment (vytick) [reply suite-native/atoms/src/AlertBox.tsx]: e5deb4e

File: suite-native/atoms/src/AlertBox.tsx (hunk 1 above)
Comment (vytick) [reply suite-native/atoms/src/AlertBox.tsx]: e5deb4e

File: suite-native/atoms/src/AlertBox.tsx [hunk 3]
```diff
@@ -1,46 +1,92 @@
...
+    backgroundColor: Color;
+    contentColor: Color;
+};
 
 type AlertBoxProps = {
-    title: string;
-    isIconVisible?: boolean;
+    title?: React.ReactNode;
+    isStandalone?: boolean;
+    variant: AlertBoxVariant;
```
Comment (PeKne) [inline suite-native/atoms/src/AlertBox.tsx]: nit: put the variant at the beginning of the props object. Required properties should be defined before the optional ones.

File: suite-native/atoms/src/AlertBox.tsx [hunk 4]
```diff
@@ -1,46 +1,92 @@
...
+
+const variantToIconName = {
+    info: 'info',
+    success: 'checkCircle',
+    warning: 'warningTriangle',
+    error: 'warningTriangle',
+} as const satisfies Record<AlertBoxVariant, IconName>;
+
+export const AlertBox = ({ title, isStandalone = false, variant = 'info' }: AlertBoxProps) => {
     const { applyStyle } = useNativeStyles();
```
Comment (PeKne) [inline suite-native/atoms/src/AlertBox.tsx:71]: nit:
```suggestion
   const {contentColor, backgroundColor} = variantToColorMap[variant];
```

Like this, you don't have to use the long version (e.g. `variantToColorMap[variant].contentColor`) multiple times.
But this is minor thing. If you don't wanna change this, I'm OK with that.

**Review by PeKne — APPROVED** [review]
Two minor nits.

File: suite-native/atoms/src/AlertBox.tsx (hunk 4 above)
Comment (vytick) [reply suite-native/atoms/src/AlertBox.tsx:71]: nice, updated in 9b4dd74f2

File: suite-native/atoms/src/AlertBox.tsx (hunk 3 above)
Comment (vytick) [reply suite-native/atoms/src/AlertBox.tsx]: changed in 9b4dd74f2

**Review by PeKne — APPROVED** [review]

**Review by Nodonisko — APPROVED** [review]

**Pull request merged**